solr/solrj/src/test/org/apache/solr/client/solrj/impl/Http2SolrClientTest.java

```diff
@@ -335,7 +327,7 @@ public void testCollectionParameters() throws IOException, SolrServerException {
 
   @Test
   public void testQueryString() throws Exception {
-    testQueryString(Http2SolrClient.class, Http2SolrClient.Builder.class);
+    super.testQueryString();
   }
```

Comment:
I *think* this isn't needed, as long as testQueryString in the superclass is marked as a test like this one.

Reply:
I initially did it that way but then found that it confused IntelliJ.